jmdaemon/jmdaemon/irc.py

```diff
@@ -200,6 +203,8 @@ def connectionMade(self):
         return irc.IRCClient.connectionMade(self)
 
     def connectionLost(self, reason=protocol.connectionDone):
+        wlog("INFO", "Lost IRC conncetion to: " + str(self.hostname)
```

Comment:
typo (s/conncetion/connection/)

Reply:
Thanks... another opportunity to practice git rebase ;)
Fixed.